.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: macos-11
     strategy:
       matrix:
-        platform: [ios, osx, tvos, watchos]
+        platform: [ios, ios, osx, tvos, watchos]
```

Comment:
ios in twice?

Reply:
Yeah, that's needed because the first one is for the integration and the second for the unit tests on ios.